UPGRADE.md

```diff
@@ -1,3 +1,9 @@
+
+# From 3.16.0 to x.x.x
```

Comment:
```suggestion
# From 3.17.0 to x.x.x
```

Reply:
If you plan to release this with 3.17.0, this has to be `# From 3.16.0 to 3.17.0`, right?

Reply:
Correct, my bad